source/whats-new.txt

```diff
@@ -69,6 +69,8 @@ Improvements in 5.1
 
 - Internal testing of GraalVM native image technology. These tests involve building
   native applications by using the GraalVM native-image tool.
+- Enhanced support for the MONGODB-OIDC authentication mechanism. For more information about
+  OIDC, see the :ref:`mongodb-oidc` section of the Enterprise Authentication Mechanisms guide.
```

Comment:
Clicking this link takes me to `/fundamentals/enterprise-auth/#std-label-mongodb-oidc` as opposed to `/fundamentals/enterprise-auth/#mongodb-oidc`, which lead to slightly different locations on the page (the latter feels like it would be the correct anchor link). Is this expected behavior?

Reply:
Since we use ref anchors to link to other pages, the `std-label` part of the link is expected. It is weird that the link brings you a little before the start of the MONGODB-OIDC section, but I'm not sure if there's a way around this - think it's just strange Snooty behavior.

Reply:
In that case, LGTM!